torch/onnx/symbolic_opset9.py

```diff
@@ -4942,8 +4942,8 @@ def rrelu(g: jit_utils.GraphContext, input, lower, upper, training, generator):
 
 @_onnx_symbolic("aten::bernoulli")
 @_beartype.beartype
-def bernoulli(g: jit_utils.GraphContext, input, generator=None, out=None):
-    if out is not None:
+def bernoulli(g: jit_utils.GraphContext, input, tensor_or_prob=None, generator=None, out=None):
```

Comment:
I don't need a `tensor_or_prob` argument in the native_functions.yaml definition. Is this new?

Reply:
I see its called `p`. I would just call it that?

(Seen)

```
- func: bernoulli_.Tensor(Tensor(a!) self, Tensor p, *, Generator? generator=None) -> Tensor(a!)
  device_check: NoCheck   # TensorIterator
  variants: method
  tags: nondeterministic_seeded
  dispatch:
    CPU, CUDA: bernoulli_
    MPS: bernoulli_mps_
  autogen: bernoulli.Tensor, bernoulli.Tensor_out

- func: bernoulli_.float(Tensor(a!) self, float p=0.5, *, Generator? generator=None) -> Tensor(a!)
  device_check: NoCheck   # TensorIterator
  variants: method
  tags: nondeterministic_seeded
  dispatch:
    CPU, CUDA: bernoulli_
    MPS: bernoulli_mps_
  autogen: bernoulli.float_out
```

Reply:
Sure, I will change it to `p` and the existing `p` to another name.